src/test/java/tests/QualifierTests.java

```diff
@@ -490,18 +473,38 @@ public void statusOrdering() {
 
         List<TurboIssue> renderedIssues = new ArrayList<>(issues);
 
-        Collections.sort(renderedIssues,
-                Qualifier.getSortComparator(model, "status", false, false));
-        
-        assertEquals(Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7), renderedIssues.stream()
-                .map(TurboIssue::getId)
-                .collect(Collectors.toList()));
-        
-        Collections.sort(renderedIssues,
-                Qualifier.getSortComparator(model, "status", true, false));
+        // test: sorting by status 
+        testSortedIssues(renderedIssues, Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7), 
+                         model, "status", false, false);
+
+        // status alias
+        testSortedIssues(renderedIssues, Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7), 
+                         model, "st", false, false);
+
+        // inverted sort order
+        testSortedIssues(renderedIssues, Arrays.asList(4, 5, 6, 7, 0, 1, 2, 3), 
+                         model, "status", true, false);
+
+    }
+    
+    /**
+     * Tests the order of TurboIssues after sorted according to a sortCriteria
+     */
+    private void testSortedIssues(List<TurboIssue> issues, List<Integer> expectedIds,
```

Comment:
The method name and comment can be a bit more specific. `assertSorted`? "Checks that TurboIssues are ordered in a particular way given some sorting criteria".